src/Microsoft.DotNet.Build.Tasks/PackageFiles/notsupported.targets

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+
+  <PropertyGroup Condition="'$(IsNotSupportedAssembly)' == 'true'">
```

Comment:
Can we call this property "AssemblyThrowsPlatformNotSupportedForAllAPIs"? Or something a little more detailed explaining what this actually implies.